slice.py

```diff
@@ -92,14 +144,24 @@ def calculateScaleAndShift(mesh, resolution):
 
 
 def scaleAndShiftMesh(mesh, scale, shift):
+    adjusted_mesh = []
     for tri in mesh:
```

Comment:
I think this for loop can be replaced with 
`adjusted_mesh = (mesh + shift) * scale`

Reply:
https://github.com/youngkiu/stl-to-voxel/blob/mesh_numpy/slice.py
I am preparing a new PR, can I review it there?